lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -98,198 +98,206 @@ def main():
 
     print(f"\n--> Start calculating calibration from run {run}, filters {filters}")
 
-    try:
-        # verify config file
-        if not os.path.exists(config_file):
-            raise IOError(f"Config file {config_file} does not exists. \n")
-
-        print(f"\n--> Config file {config_file}")
-
-        # verify input file
-        file_list=sorted(Path(f"{base_dir}/R0").rglob(f'*{run}.{sub_run:04d}*'))
+    # verify config file
+    if not os.path.exists(config_file):
+        print(f"Config file {config_file} does not exists. \n")
+        sys.exit(1)
+
+    print(f"\n--> Config file {config_file}")
+
+    # verify input file
+    file_list=sorted(Path(f"{base_dir}/R0").rglob(f'*{run}.{sub_run:04d}*'))
+    if len(file_list) == 0:
```

Comment:
`if not file_list:`

Reply:
Not really an improvement I'd say. `len(file_list) == 0` is very explicit about what is checked and since file_list is always a  list and not something else (i.e. None), there is no harm in being explicit